### DISCUSSION TITLE
UFW firewall

### DISCUSSION THREAD

**Discussion by knightflyer** [ltsp/ltsp#586] [General] [answered]
Mint 20.2 UMA all is running well, except when I enable UFW on the server.  I have allowed connection from all LAN PCs on 192.168.1.0/24 enabled and one outside IP cloud PBX server enabled everything is fine and dandy except clients will not see the server when UFW is enabled (they won't boot) if I disable UFW on the server they boot immediately. I can then enable UFW and all is okay.  This is a problem.
Do I need to enable something else? Like ipxe?  Any ideas on how to make the magic happen?

**Comment by alkisg** [comment, marked as answer]
I'm not using ufw; if anyone is, feel free to comment.
That said, this worked for me, while I was using the internal LTSP subnet:

```shell
ufw enable
ufw allow from 192.168.67.0/24
ufw allow proto udp from 0.0.0.0/0 to 0.0.0.0/0 port 67
```

**Reply by knightflyer** [reply]
This solve the issue. Thanks alkisg
